packages/orchestration/src/examples/send-anywhere.contract.js

```diff
@@ -43,10 +46,16 @@ const contract = async (
 
   const creatorFacet = prepareChainHubAdmin(zone, chainHub);
 
+  // UNTIL https://github.com/Agoric/agoric-sdk/issues/9066
+  const logNode = E(privateArgs.storageNode).makeChildNode('log');
+  /** @type {(msg: string) => Vow<void>} */
+  const log = msg => vowTools.watch(E(logNode).setValue(msg));
```

Comment:
It's expected that the logs of all invocations will write to the same node?

Reply:
Until #9066, yes.